.evergreen/generate_evergreen_tasks.js

```diff
@@ -507,15 +507,27 @@ BUILD_VARIANTS.push({
 const oneOffFuncs = [
   { func: 'run custom snappy tests' },
   {
+    name: 'run-bson-ext-integration',
+    func: 'run bson-ext test',
+    vars: {
+      NODE_LTS_NAME: LOWEST_LTS,
+      TEST_NPM_SCRIPT: 'check:test'
+    }
+  },
+  {
+    name: 'run-bson-ext-unit',
     func: 'run bson-ext test',
     vars: {
-      NODE_LTS_NAME: LOWEST_LTS
+      NODE_LTS_NAME: LOWEST_LTS,
+      TEST_NPM_SCRIPT: 'check:unit'
     }
   }
 ];
 
 const oneOffFuncAsTasks = oneOffFuncs.map(oneOffFunc => ({
-  name: `${oneOffFunc.func.split(' ').join('-')}`,
+  name: `${
```

Comment:
rather than patching the logic here, what if we just gave the other instances of oneOffFuncs a `name` property?  Having a "name" makes more sense to me anyways and is one less thing to maintain.

Reply:
sure, done